extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -75,6 +76,12 @@ define('ext.wikia.adEngine.adContext', [
 				context.opts.pageFairDetection = true;
 			}
 		}
+		// SourcePoint recovery
+		if (!noExternals && context.opts.sourcePointRecovery !== false) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)